wordpress-mobile/WordPress-Android
# Add initial Quick Start Task card item implementation to the improved My Site fragment

Fixes #13584

This PR builds upon #13724 and #13725 and it basically introduces the check icon for done tasks, click handling and overall styling for the card.

Under the hood, the dummy classes were extracted and renamed and the layout is now using a `MaterialCardView` so we can take advantage of its built-in checkable states. This was customized quite a bit, first by tinting the check icon so it matches the color used by the parent card progress bar and also by introducing custom foreground and ripple colors, so we can fine tune what the card looks like when it's clicked and done, both in light and dark modes. We're also using a `ShapeableImageView` to control the corner radius of the illustrations, which matches the one for the card.

https://user-images.githubusercontent.com/830056/104662064-ba98c280-56a8-11eb-91d6-92ff065ac13d.mp4

### To test

Make sure all unit tests in `MySiteViewModelTest` pass. 

Additionally:

1. Turn on the Improved My Site flag and restart the app.
1. Go to the My Site screen.
1. Notice the sample Quick Start Task cards.
1. Make sure nothing looks out of place.
1. Click on a task card that is not done.
1. Notice the ripple effect.
1. Notice a snackbar with the task id.
1. Click on a task card that is done.
1. Notice how nothing happens.

---

PR submission checklist:
